features/element-hiding/index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Element Hiding</title>
+    <script src='./main.js' defer></script>
+    <style type="text/css">
+        .container {
+            width: 400px;
+            height: 200px;
+            background-color: rgb(249, 251, 251);
+            color: rgb(88, 104, 113);
+            font-size: 16px;
+            line-height: 28px;
+            padding: 20px;
+            margin: 25px;
+        }
+        #results-table {
+            border-collapse: collapse;
+            width: 70%;
+        }
+        #results-table td, #results-table th {
+            border: 1px solid #ddd;
+            padding: 8px;
+        }
+        #results-table th {
+            padding-top: 12px;
+            padding-bottom: 12px;
+            text-align: left;
+            background-color: #f2f2f2;
+            color: black;
+        }
+        .spacer {
+            height: 1000px;
+        }
+    </style>
+</head>
+<body>
+    <p><a href="../../">[Home]</a> ↣ [Features] ↣ <strong>[Element Hiding Test Page]</strong></p>
+
+    <p>Test page for element hiding, a feature used to hide DOM elements. Tests take a few seconds to run.</p>
+
+    <table id="results-table">
+        <tr><th>Description</th><th>Outcome</th></tr>
+    </table>
+
+    <div class="spacer">
```

Comment:
nit, but instead of this you could just say that below will be debug data like I did in https://privacy-test-pages.glitch.me/privacy-protections/request-blocking/ - makes debugging slightly easier if you have to load the page multiple times